### DISCUSSION TITLE
Is it possible to pin fields when adding a filter

### DISCUSSION THREAD

**Discussion by Dumpstar56** [directus/directus#12959] [Feature Requests]
I have several tables with a lot of fields and relations, and I frequently have to add filters for specific fields at the bottom of the list of fields or within the nested relations when adding a filter. Can I somehow pin some fields to the top of the list when adding a filter without changing the layout or order of the fields in the collection?

**Comment by rijkvanzanten** [comment]
There's currently no way to pin fields like that. I'll convert this Q&A to be the feature request for such a feature 👍🏻

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
